.github/workflows/pull_request_verification.yml

```diff
@@ -21,7 +21,13 @@ jobs:
       - name: Grant execute permission for gradlew
         run: chmod +x gradlew
       - name: Check lint
-        run: ./gradlew lintDebug
+        run: |
+          gpg --quiet --batch --yes --decrypt --passphrase="$FIREBASE_SECRET" \
+          --output /home/runner/work/Picover/Picover/app/services.tar **/services.tar.gpg
+          tar xvf /home/runner/work/Picover/Picover/app/services.tar
```

Comment:
shouldn't this by finally extracted to some file, which name is google-services.json and it is in the Picover/app/ directory ? As far as i know, it could be a problem to find it now during the build process

Reply:
I used way from the link which you sent in another comment